### ISSUE TITLE
Mailing lists

### ISSUE THREAD

**Issue by davidar** [ipfs/archives#17]
http://news.gmane.org/

Cc: @cryptix

**Comment by jbenet** [comment]
Yes, i was thinking about this a few weeks ago. do we have a nice HTML-only viewer?

Also #18

**Comment by davidar** [comment]
There's [Loom](http://loom.gmane.org/) ([example](http://news.gmane.org/gmane.linux.kernel)). The HTML conversion seems to be performed server-side, but I imagine that could be done in JS without too much difficulty.

/me defers to @diasdavid

**Comment by diasdavid** [comment]
There are plenty of rendering engines for browser web applications, also porting one a JS server side one shouldn't take a load of time, but if it wasn't written in JS from the start, might expensive.

One idea would be to store with the objects with an extra merkle-link to a HTML rendered version -> This might actually be pretty neat, only having to one render for each page version at max.

**Comment by jbenet** [comment]
yeah that could work too. i think someone will make a viewer in pure js. we just need to provide the data in json first.

**Comment by davidar** [comment]
https://lars.ingebrigtsen.no/2016/07/28/the-end-of-gmane/ :(
